src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -73,9 +85,46 @@ public class DevBoxInstance : IComputeSystem
         get; private set;
     }
 
+    public IEnumerable<ComputeSystemProperty> Properties

```

Comment:
```C#
public IEnumerable<ComputeSystemProperty> Properties  { get; set; } = new List<ComputeSystemProperty>();
```

Then you don't need to create it in the constructor.

Reply:
Resolved.